test/integration/targets/inventory_docker_machine/playbooks/teardown.yml

```
@@ -0,0 +1,8 @@
+---
+- hosts: 127.0.0.1
+  connection: local
+  vars:
+    dm_machine_name: "{{ lookup('env', 'DM_MACHINE_NAME') }}"
+  tasks:
+    - name: Request Docker Machine to terminate the Digital Ocean droplet
+      shell: "docker-machine rm {{ dm_machine_name }}"
```

```suggestion
      shell: "docker-machine rm -f {{ dm_machine_name }}"
```
(`-y` would also work for me, but `-f` should be safer)

---

Done.